app/serializers/api/v1/offer_summary_serializer.rb

```diff
@@ -83,5 +99,18 @@ def goodcity_user?
       User.current_user.present?
     end
 
+    def restrict_payload_for_offers_package?
+      !options[:restrict_payload_for_offers_package]
```

Comment:
Doesn't make sense to me that `restrict_payload_for_offers_package?` would equal to the negative `(NOT) !options[:restrict_payload_for_offers_package]`